src/components/ParticipantInfo/ParticipantInfo.tsx

```diff
@@ -121,6 +134,28 @@ const useStyles = makeStyles((theme: Theme) =>
         fontSize: '0.75rem',
       },
     },
+    switchedOffMessage: {
```

Comment:
This text could also have a shadow to make it more readable. Without it, you wouldn't be able to read white text on a white background. I don't know much about text-shadow, but something like this seemed ok: `textShadow: '0 0 3px rgba(0, 0, 0, 0.7)',`

More info here: https://css-tricks.com/almanac/properties/t/text-shadow/